Bottom: 929bdd4d6803e46bb028150818fe188cb32a0e8c
Top:    db510e9299cb4ef1e3799b97e5c9e4a868eadbfd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-21 23:44:33 +0000

Allow running of action scripts when running PCB as an exporter


---

diff --git a/src/main.c b/src/main.c
index e5d93eb..992801c 100644
--- a/src/main.c
+++ b/src/main.c
@@ -991,12 +991,6 @@ main (int argc, char *argv[])
       LayerStringToLayerStack (Settings.InitialLayerStack);
     }
 
-  if (gui->printer || gui->exporter)
-    {
-      gui->do_export (0);
-      exit (0);
-    }
-
   /*    FIX_ME
      LoadBackgroundImage (Settings.BackgroundImage); */
 
@@ -1029,6 +1023,12 @@ main (int argc, char *argv[])
       hid_parse_actions (Settings.ActionString, 0);
     }
 
+  if (gui->printer || gui->exporter)
+    {
+      gui->do_export (0);
+      exit (0);
+    }
+
 #if HAVE_DBUS
   pcb_dbus_setup();
 #endif
